docs/Docs.md

```diff
@@ -188,12 +188,23 @@ ProbeInterval = BaseInterval * (NSA+1)
 
 nack은 SWIM 프로토콜에 해당하지는 않는다. 그렇지만 Lifeguard가 자기자신에게 돌아오지 않는 메세지를 고려하기 때문에 nack이 유용하게 사용될 수 있다. 예를 들어보면 네트워크 내에 A, B, C, D가 있는데 A가 D에게 메세지를 보낸다고 하자. 이 때 아무런 응답이 다시 A에게 돌아오지 않을 때 nack을 사용하지 않는다면 B, C, D 중에 누가 문제인지 알 수 없다. 반면에 nack을 사용한다고 했을 때 nack이 B, C로 부터 돌아오면 A, B, C 끼리는 통신이 잘 되는 것을 확인할 수 있고 문제의 원인은 D에 있을 가능성이 높아진다. 또 nack이 B, C 로부터 잘 돌아오지 않는다면 A가 isolated 되있을 가능성이 높다.
 
+probe timeout의 80%까지 ack를 받지 못할경우, 자신에게 indirect ping을 보낸 노드(A)에게 nack 메세지를 보낸다. 만약, A에게 nack을 보내고 난 후 ack 메세지를 받았을 경우 다시 A에게 ack 메세지를 보낸다. 이것 또한 indirect probe가 성공되었다고 간주한다.
+
 ###### L2: Dynamic Suspicion Timeouts: "Dogpile"
 
 기본적인 원리는 L1과 비슷하게 자신에게 돌아오지 않는 메세지가 있을 때 suspection timeout을 늘리게된다. 왜냐하면 메세지가 잘 들어오지 않는 다는 것은 자기 자신이 느리거나 다른 노드들과 isolated 되있을 가능성이 크다는 것이기 때문에 suspect 노드로 부터 alive 메세지가 timeout을 방지하기 위해서 timeout을 늘린다. 그래서 방식을 정리하면 다음과 같다. 최초로 노드는 큰 suspicion timeout 값을 가지고 시작한다. 그리고 자기 자신이 많은 메세지를 받을 수록 suspicion timeout을 낮춘다.
 
-###### L3: More Timely Refutation: "Buddy System"
+<p align="center">
+<img src="../images/[lifeguard]SuspicionTimeout.png" width="450px" height="100px">
+</p>
 
```

Comment:
I think image is broken, please check again :+1:

Reply:
I checked!